websites/api-docs-smoke-test/.gitignore

```diff
@@ -0,0 +1 @@
+@commercetools-docs
```

Comment:
The gitignore should be autogenerated, located directly next to the generated file and cover only this exact file.  

That way it is not forgotten, does not interfere with other intentional .gitignore 's and still allows intended component shadowing in a given website.

Reply:
We could also just commit the file in VSC, it shouldn’t be a big of a problem.
I’ll add the gitignore